troubleshooting/fuzzy-window-rendering.md

```diff
@@ -0,0 +1,23 @@
+# Fuzzy window rendering on external 4K display
```

Comment:
I think this should be in docs.

Reply:
Do you think FAQ section is a good choice?

Reply:
> Do you think FAQ section is a good choice?

Yes.